SPECS/systemd/systemd.spec

```diff
@@ -29,6 +29,7 @@ Patch7:         serve-stale-0001-resolved-added-serve-stale-feature-implementati
 Patch8:         serve-stale-0002-resolved-Initialize-until_valid-while-storing-negati.patch
 BuildRequires:  audit-devel
 BuildRequires:  cryptsetup-devel
+BuildRequires:  dbus
```

Comment:
Does the build actually requires these additional packages, or just running %check? Is there a way to make these requirements conditional on whether check is being invoked? (I see some other specs wrap such requirements with a conditional on `with_check`.)